### PR TITLE
ODP-2017 Livy CVE's reconciliation from ODP-3.2.3.2-3 to ODP-main

### PR DESCRIPTION
**Pull request by senthh**
## What changes were proposed in this pull request?

Below commits have beed backported,

[SNAPSHOT to 3.2.2.0-1095](https://github.com/acceldata-io/livy/commit/38b7a03ebe12a2c01355a88a8e5003234e82df22)

[use ODP version instead of apache](https://github.com/acceldata-io/livy/commit/ee434b1d2897deb92fae9201861a8a7ef108844e)

[Release 3.2.3.2-2](https://github.com/acceldata-io/livy/commit/847967e3183903829db1fb35aec665e9046652e2)

[Release 3.2.3.2-3](https://github.com/acceldata-io/livy/commit/efea81f23667eaab345aae24d778d841b596f26c)

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.livy</groupId>
     <artifactId>livy-main</artifactId>
-    <version>0.7.1.3.2.3.0-SNAPSHOT</version>
+    <version>0.7.1.3.2.3.2-3</version>
```
Comment (prabhjyotsingh) [inline api/pom.xml:23]: This still looks wrong, it should be 0.7.1.3.2.3.0-SNAPSHOT.
Can you revert these changes?

Comment (senthh) [reply api/pom.xml:23]: As per commits Release 3.2.3.2-2 and Release 3.2.3.2-3, 0.7.1.3.2.3.0-SNAPSHOT is getting changed
